pkg/kubelet/kuberuntime/kuberuntime_container_test.go

```diff
@@ -61,7 +61,7 @@ func TestRemoveContainer(t *testing.T) {
 	err = m.removeContainer(containerId)
 	assert.NoError(t, err)
 	// Verify container log is removed
-	expectedContainerLogPath := filepath.Join(podLogsRootDirectory, "12345678", "foo_0.log")
+	expectedContainerLogPath := filepath.Join(podLogsRootDirectory, "12345678", "/foo/foo_0.log")
```

Comment:
Use `Join` instead of `/`.

"foo", "0.log"

Reply:
done